app/repositories/section_repository.rb

```diff
@@ -9,10 +9,6 @@ def fetch(section_id)
     end
   end
 
-  def initialize(manual:)
-    @manual = manual
-  end
-
```

Comment:
I've updated the commit subject line to "Move manual param from SectionRepository#initialize to #fetch".